Server/mods/deathmatch/logic/CPerfStat.PlayerPacketUsage.cpp

```diff
@@ -408,11 +408,11 @@ void CPerfStatPlayerPacketUsageImpl::OutputTimeSpanBlock(CPerfStatResult* pResul
             const CTimeSpan* p = timeSpanList[i];
             const CTopValue& value = (p->prev.data[0].uiPktsPerSec > p->acc.data[0].uiPktsPerSec) ? p->prev.data[iTopPos] : p->acc.data[iTopPos];
 
-            row[c++] = value.uiPktsPerSec ? SString("%d", value.uiPktsPerSec) : "-";
+            row[c++] = value.uiPktsPerSec ? std::to_string(value.uiPktsPerSec) : "-";
             if (bDetail)
             {
-                row[c++] = value.uiBytesInPerSec ? SString("%d", value.uiBytesInPerSec) : "-";
-                row[c++] = value.uiBytesOutPerSec ? SString("%d", value.uiBytesOutPerSec) : "-";
+                row[c++] = value.uiBytesInPerSec ? std::to_string(value.uiBytesInPerSec) : "-";
```

Comment:
I see you want to use a feature from C++20 and that's great. But in this case we have an unwanted string copying. But you can combine C++20 with SString by adding a move constructor to it. The same for above and below.

Reply:
So `SString(std::to_string(...))`?

Reply:
In theory, you can leave it as is. But you need to modify `SString` and add a move constructor. Like 
`SString(std::string&& strText) : std::string(std::move(strText)) {}`